app/stacks/MasterDetailStack/ModalContainer.tsx

```diff
@@ -25,8 +28,21 @@ const styles = StyleSheet.create({
 });
 
 export const ModalContainer = ({ navigation, children, theme }: IModalContainer): JSX.Element => {
+	const keyboardHeight = useKeyboardHeight();
 	const { height } = useWindowDimensions();
+	const { isLandscape } = useOrientation();
 	const modalHeight = sharedStyles.modalFormSheet.height;
+
+	let heightModal: number;
+
+	if (modalHeight > height) {
+		heightModal = height;
+	} else if (isLandscape && isAndroid && keyboardHeight > 0) {
```

Comment:
I'm not sure about testing landscape here.
The modal could be big enough to break the usage if it's an Android tablet on portrait, right?

Reply:
I refactor this logic